components/brave_wallet_ui/utils/string-utils.test.ts

```diff
@@ -43,3 +43,12 @@ describe('Checking URL ends with a valid icon extension', () => {
     expect(isValidIconExtension('https://test.com/')).toEqual(false)
   })
 })
+
+describe('Check toDouble values', () => {
+  test('Value with a USD symbol, should remove the USD symbol', () => {
+    expect(toDouble('$1,689.16')).toEqual('1,689.16')
+  })
+  test('Value with a BRAZ symbol, should remove the BRAZ symbol', () => {
```

Comment:
Update test description, this is testing that the Euro symbol is being removed.

Reply:
Done!